dunstctl

```diff
@@ -29,8 +29,10 @@ show_help() {
 	                                    notification with given ID.
 	  history-rm [ID]                   Remove the notification from
 	                                    history with given ID.
-	  is-paused                         Check if dunst is running or paused
-	  set-paused [true|false|toggle]    Set the pause status
+	  is-paused                         Check if pause level is > 0
+	  set-paused [true|false|toggle]    Set the pause status	  
+	  get-paused-level                  Get the current pause level
```

Comment:
I would rename `get-paused-level` and `set-paused-level` to `get-pause-level` and `set-pause-level` (without the d's) for more correct grammar.